plugins/connection/docker.py

```diff
@@ -308,11 +330,32 @@ def put_file(self, in_path, out_path):
                 count = ' count=0'
             else:
                 count = ''
-            args = self._build_exec_cmd([self._play_context.executable, "-c", "dd of=%s bs=%s%s" % (out_path, BUFSIZE, count)])
+            if getattr(self._shell, "_IS_WINDOWS", False):
+                cmd = '''& { $strm = [Console]::OpenStandardInput(1024)
```

Comment:
If you use `_encode_script` as per below you can skip the `& { ... }` entirely and just run the command as is.

Reply:
@jborean93 changed the code as advised, but I receive an `invalid payload`

![image](https://user-images.githubusercontent.com/14177833/117200240-8ed25180-adeb-11eb-8dff-657ab3d64db1.png)


![image](https://user-images.githubusercontent.com/14177833/117200226-89750700-adeb-11eb-950b-d0c1e91cdb66.png)

Reply:
I'm not sure what is happening here, the error you've shared is something that is thrown by the [bootstrap_wrapper.ps1](https://github.com/ansible/ansible/blob/devel/lib/ansible/executor/powershell/bootstrap_wrapper.ps1) which is used when executing a module. The code you have here is for copying a file to the docker container and shouldn't be calling the bootstrap_wrapper code.